dashboard/config/scripts/edit-code.script

```diff
@@ -0,0 +1,32 @@
+id '3'
+title 'Edit Code'
+description ""
+wrapup_video 'hoc_wrapup'
+
```

Comment:
I think it would be nice to explicitly include `hidden "true"` on scripts we want to be hidden so people don't have to think about what the default is